docs/creating_component_code_review.md

```diff
@@ -47,8 +49,15 @@ bridge = Bridge(...)
 status = bridge.status()
 ```
 
-### 5. Limit platforms included in initial pull request
-Large pull requests mean there is a larger chance of finding problems that need to be addressed, and more code that needs to be reviewed between every requested change. If your new component will have multiple platforms, try and limit your initial pull request to a single platform. Once the initial component is merged, you can submit additional PRs for the remaining platforms. This allows reviewers to sign off on smaller chunks of code one at a time, and lets us get your new feature in sooner. Pull requests containing large code dumps will not be a priority for review.
+### 5. Limit initial pull request to Minimum Viable Product (MVP)
+Large pull requests mean there is a larger chance of finding problems that need to be addressed, and more code that needs to be reviewed between every requested change. Limit the initial pull request to the minimum functionality needed for someone to get value out of the integration. Once the initial pull request is merged, you can submit additional PRs for the remaining features. This allows reviewers to sign off on smaller chunks of code one at a time, and lets us get your new integration/features in sooner. **Pull requests containing large code dumps (> 800 line adds excluding tests) will not be a priority for review and may be closed.** Limit MVPs to the following:
+- A single platform
+- No custom services 
+- Loaded via `configuration.yaml` _or_ config flow (if `configuration.yaml` cannot be used)
+- No other features outside of supporting the single platform
 
 ### 6. Event names
 Prefix component event names with the component name itself. For example, use `netatmo_person` instead of `person` for the `netatmo` component.
+
+### 7. Tests
+Strongly consider adding tests for your component to ensure it works correctly. Pull Requests with tests are treated with a higher priority than those without.
```

Comment:
Are they? Has this been explicitly stated previously?

Reply:
If it is not a requirement, it should not be on the checklist. "Okay, I strongly considered adding tests. Check."

Personally, I actually tend to favor reviewing PRs without tests since tests are very hard to review :)

Reply:
Probably not, but wouldn't it make sense from a QA perspective to require tests in general?

Reply:
I don't prioritize based on tests. I prioritize known contributors first, and then I pick from the back of my notifications, to start with the oldest PRs.

Reply:
Just remove the second sentence.

Reply:
- We can't require tests, but I think we should be encouraging it.  I was hoping to we could create incentive by saying they are treated with higher priority.
- I find it actually easier to review PRs that have tests because that gives me more confidence the changes actually work.  If the author put the effort into writing tests, I think that should be prioritized over someone who didn't, but understand if we're not ready to do this as an organization.

Reply:
I will prioritize PRs that have tests. I can update LabelBot to add a `has-tests` label too.

Reply:
I love that idea.  Then reviewers can prioritize if they wish.  We may also want to consider updating the `small-pr` logic to exclude tests, so it's not a "punishment" to write tests where it otherwise may have been small.